contrib/local-environment/keycloak/wait-for-keycloak.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+# wait-for-keycloak.sh
+
+set -e
+
+cmd="$@"
+
+function test_http_code(){
```

Comment:
This was causing unexpected `(` issues for me, should it just be `test_http_code() {`?

Reply:
Got rid of all scripts.